### PR TITLE
2913: jQuery ui touch punch altered to enable vertical scroll

### PR DESCRIPTION
**Pull request by KristianB14**
https://platform.dandigbib.org/issues/2913

### CODE REVIEW

File: modules/ting_search_carousel/js/jquery.ui.touch-punch.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
+/*!
```
Comment (kasperg) [inline modules/ting_search_carousel/js/jquery.ui.touch-punch.js]: This is an external dependency. Please handle it using `drush make`.

Comment (kasperg) [inline modules/ting_search_carousel/js/jquery.ui.touch-punch.js]: Please do not include third party code in the codebase. They should be downloaded using `drush make`.

File: modules/ting_search_carousel/js/jquery.ui.touch-punch.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+   */
+  function simulateMouseEvent (event, simulatedType) {
+
+    // Ignore multi-touch events
+    if (event.originalEvent.touches.length > 1) {
+      return;
+    }
+
+    // @COMMENTED OUT - line is moved to if-statement below
+    //event.preventDefault();
```
Comment (kasperg) [inline modules/ting_search_carousel/js/jquery.ui.touch-punch.js]: If we need to alter third party code then please do it using patches. Not by hacking directly in the source code.

**Comment by DDBdrift** [comment]
https://platform.dandigbib.org/issues/2913 | Closed

**Pull request closed without merging**